guides/v2.2/performance-best-practices/software.md

```diff
@@ -26,7 +26,7 @@ See [Magento 2.2.x technology stack requirements]({{ page.baseurl }}/install-gde
 
 Operating system configurations and optimizations are similar for Magento as other high-load web applications. As the number of concurrent connections handled by the server increases, the number of available sockets can become fully allocated. The Linux kernel supports a mechanism to "reuse" and "recycle" TCP connections. Be aware that more aggressive recycling than re-use may cause issues on the load balancers. To enable these kernel settings, set the following values in `/etc/sysctl.conf`: 
 
-```
+```bash
```

Comment:
bash is for a single command line command. The CSS adds a "$ " before the first character on the first line only.

These are lines in a file, so the original three backticks was correct. Had this been a pair of commands, each line would need to be wrapped with the bash code gates individually.

Reply:
Ah thanks.